font_manager.py

```diff
@@ -0,0 +1,17 @@
+from pygame import font as _font
+from sys import platform as _platform
+
+class FontManager(object):
+    def __init__(self):
+        self._fonts = {}
+        if _platform == "darwin":
```

Comment:
Yes, there are lots of open fonts we could use.

New plan: Add font directory and config line; makes for easy changes, at least during development if not for release.